internal/manifests/collector/collector.go

```diff
@@ -32,7 +32,6 @@ func Build(params manifests.Params) ([]client.Object, error) {
 		params.Log.V(5).Info("not building sidecar...")
 	}
 	manifestFactories = append(manifestFactories, []manifests.K8sManifestFactory{
-		manifests.Factory(ConfigMap),
```

Comment:
Why are we not using the ConfigMap Factory?

Reply:
> Cannot use []*corev1.ConfigMap as the type client.Object Type does not implement 'client.Object'
We have to call it and then add it into the `resourceManifests` since we return multiple configmaps.